src/BreathOfTheWild/Mods/MasterModeOptions/rules.txt

```diff
@@ -0,0 +1,111 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Master Mode Options"
+path = "The Legend of Zelda: Breath of the Wild/Mods/Master Mode Options"
+description = Allows you to pick and choose which of Nintendo's tweaks |are applied to Master Mode (Does nothing in Normal Mode).|A reboot is required to apply changes made while the game |is running. ||Logic and patching by Torphedo and Fleevoid, with lots of |help from Exzap and leoetlino.

```

Comment:
Is the new line insertion in the middle of sentences required? It should preferably only be used at the end of a line. Line wrapping by Cemu is a bit messy sometimes, but it's probably better to have it scale with the window size still.

Since the description should be try to kept short and concise, the credits could maybe be changed to something like:
```
Made by Torphedo and Fleevoid, with help from Exzap and leoetlino.
```
This makes it more like the other graphic pack credits.